driver-core/src/main/com/mongodb/operation/MapReduceHelper.java

```diff
@@ -26,19 +27,19 @@ static MapReduceStatistics createStatistics(final BsonDocument result) {
     }
 
     private static int getInputCount(final BsonDocument result) {
-        return result.getDocument("counts").getNumber("input").intValue();
```

Comment:
These changes are to prevent the driver from failing when no statistics are present in the reply.  Without this change, we can't run mapReduce operations at all.

Reply:
Ack